src/ToggleGroup/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { ToggleGroup, Toggle } from '../';
+import { text, boolean, select } from '@storybook/addon-knobs';
+
+storiesOf('ToggleGroup', module).add(
+  'ToggleGroup with Title',
+  () => (
+    <ToggleGroup
+      label={text('label', 'This is my label')}
+      vertical={boolean('vertical', false)}
+      toggleSize={select('toggleSize', ['sm', 'md', 'lg'], 'md')}
```

Comment:
This appears to change only the label size.

Reply:
Done